src/BlockTypesController.php

```diff
@@ -239,6 +239,7 @@ protected function get_block_types() {
 			'ProductCategories',
 			'ProductCategory',
 			'ProductCollection',
+			'NoResults',
```

Comment:
May be worth adding a prefix suggesting this block belongs to Product Collection. What do you think?

```suggestion
			'ProductCollectionNoResults',
```

Reply:
Made the changes in [0a4395a](https://github.com/woocommerce/woocommerce-blocks/pull/11783/commits/0a4395a839aaddc4d4cdfdc0bf3e656f8244f53e)